apps/ranking/test/connectMongoose.ts

```diff
@@ -14,6 +14,7 @@ declare global {
 
 export async function connectMongoose() {
   jest.setTimeout(20000);
+  mongoose.set('strictQuery', false);
```

Comment:
Set this on connect mongoose on the real code as well

Reply:
https://github.dev/pjonatansr/ccsseraphini/blob/b3c830caf9953ffd678e059d085d9408d7cebd8d/apps/bot/src/mongodb.ts#L14

Já tinha colocado lá, mas fiz um ammend colocando no arquivo de teste do repo do bot